cmd/crates/soroban-spec-typescript/fixtures/test_custom_types/README.md

```diff
@@ -6,7 +6,7 @@ This library was automatically generated by Soroban CLI using a command similar
 
 ```bash
 soroban contract bindings ts \
-  --rpc-url https://rpc-futurenet.stellar.org:443 \
+  --rpc-url https://rpc-futurenet.stellar.org:443/soroban/rpc \
```

Comment:
Erm, I'm not sure where this change came from. Is it accurate?

Reply:
Probably not accurate? The /soroban/rpc endpoint is a quickstart endpoint, not a soroban-rpc endpoint.

Reply:
I figured, especially because of this code: https://github.com/stellar/soroban-tools/blob/8906a53d478f36ef434bc97b596b9bc69f014de0/cmd/soroban-cli/src/commands/network/mod.rs#L190-L197

but I don't see where this fixture is getting replaced from :thinking: I guess I'll just manually update it to unblock, as long as the E2E tests pass.

Reply:
Is it in a Rust doc somewhere?

Reply:
Ah it gets overwritten anyway, but it's a README so I guess it's fine.

Reply:
Well, it's a readme that's wrong, so not really fine. @chadoh @willemneal Any idea where the path is coming from?